### PR TITLE
Adding down and side specials 

### PR DESCRIPTION
**Pull request by KaiYi-David-Zhang**

### CODE REVIEW

File: PlatformFighter/Assets/Scripts/PlayerController.cs [hunk 1]
```diff
@@ -28,18 +33,19 @@ public class PlayerController : MonoBehaviour
     public float verticalAxis;
 
     // protected fields
-    protected bool hasControl = true;
-    protected bool isGrounded = false;
+    public bool hasControl = true;
```
Comment (99inha) [inline PlatformFighter/Assets/Scripts/PlayerController.cs:36]: any reason why these are public?

File: PlatformFighter/Assets/Scripts/Rectangle.cs [hunk 2]
```diff
@@ -19,14 +19,12 @@ protected override void jab()
     {
         UnityEngine.Debug.Log("rectangle is jabbing");
         anime.setAnimator(AnimeState.Jab);
-        lagTime = 0f;
```
Comment (99inha) [inline PlatformFighter/Assets/Scripts/Rectangle.cs:22]: why are all the lag times deleted?

Comment (KaiYi-David-Zhang) [reply PlatformFighter/Assets/Scripts/Rectangle.cs:22]: the lag times are now all in the animation, at the end of all animation the attackLag function will be called, adding the lag time there

File: PlatformFighter/Assets/Scripts/PlayerController.cs (hunk 1 above)
Comment (KaiYi-David-Zhang) [reply PlatformFighter/Assets/Scripts/PlayerController.cs:36]: forgot to change it back

Comment (99inha) [reply PlatformFighter/Assets/Scripts/PlayerController.cs]: okay welp I'll change it back later

**Pull request merged**